src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -145,17 +146,30 @@ struct patcher
   void
   patch_ctrl48(uint32_t* bd_data_ptr, uint64_t patch)
   {
-    constexpr uint64_t DDR_AIE_ADDR_OFFSET = ((uint64_t)(0x80000000));
+    constexpr uint64_t ddr_aie_addr_offset = 0x80000000;
```

Comment:
What is the significance of the values that are hardcoded in this function and in patch_shim48?   
A few comments in the right places would be helpful.